docs/Quantiles.ipynb

```diff
@@ -0,0 +1,562 @@
+        }
+      ],
+      "source": [
+        "multiple_quantiles_actual = df.select(\n",
+        "    [pl.col(\"AGE\").fill_null(mean_age).quantile(q).alias(f\"Quantile_{q}\") for q in [0.2, 0.4, 0.6, 0.8]]\n",
+        ").collect()\n",
+        "multiple_quantiles_actual"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 17,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 128
+        },
+        "id": "JuliF45iXVbO",
+        "outputId": "84a800f0-c323-4408-a66d-1f0395eede64"
+      },
+      "outputs": [
+        {
+          "data": {
+            "text/html": [
+              "<div><style>\n",
+              ".dataframe > thead > tr,\n",
+              ".dataframe > tbody > tr {\n",
+              "  text-align: right;\n",
+              "  white-space: pre-wrap;\n",
+              "}\n",
+              "</style>\n",
+              "<small>shape: (1, 4)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>Quantile_0.2</th><th>Quantile_0.4</th><th>Quantile_0.6</th><th>Quantile_0.8</th></tr><tr><td>i64</td><td>i64</td><td>i64</td><td>i64</td></tr></thead><tbody><tr><td>20</td><td>32</td><td>58</td><td>72</td></tr></tbody></table></div>"
+            ],
+            "text/plain": [
+              "shape: (1, 4)\n",
+              "┌──────────────┬──────────────┬──────────────┬──────────────┐\n",
+              "│ Quantile_0.2 ┆ Quantile_0.4 ┆ Quantile_0.6 ┆ Quantile_0.8 │\n",
+              "│ ---          ┆ ---          ┆ ---          ┆ ---          │\n",
+              "│ i64          ┆ i64          ┆ i64          ┆ i64          │\n",
+              "╞══════════════╪══════════════╪══════════════╪══════════════╡\n",
+              "│ 20           ┆ 32           ┆ 58           ┆ 72           │\n",
+              "└──────────────┴──────────────┴──────────────┴──────────────┘"
+            ]
+          },
+          "execution_count": 17,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "multiple_quantiles = context.query().select(\n",
+        "    [pl.col(\"AGE\").fill_null(mean_age).dp.quantile(q, candidates).alias(f\"Quantile_{q}\") for q in [0.2, 0.4, 0.6, 0.8]]\n",
+        ").release().collect()\n",
+        "multiple_quantiles"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "fYAFhv6vXVbO"
+      },
+      "source": [
+        "Here we computed some sample quantiles and the differentially private values are relatively close to the actual values. Notice that we also specified a different alias for each column. Polars uses the original column name on derived columns, so aliases are needed to distinguish these columns.\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "zA0D8TeJXVbO"
+      },
+      "source": [
+        "## Breaking Down the Quantile Method"
```

Comment:
My sense is that we want to introduce _how_ to get calculate quantiles early on, but this discussion might be reserved for a separate notebook in the Theory section?